src/main/java/seedu/address/logic/parser/meeting/CancelParser.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.parser.meeting;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.meeting.CancelCommand;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class CancelParser {
```

Comment:
Shouldn't we stick with the naming convention of XYZCommandParser rather than XYZParser?